ludwig/models/llm.py

```diff
@@ -159,15 +164,21 @@ def output_feature_decoder(self) -> OutputFeature:
     def initialize_adapter(self):
         """If an adapter config is provided, we want to wrap the model with a PEFT model for fine-tuning."""
         if self.config_obj.adapter:
-            from peft import get_peft_model
+            if self.config_obj.trainer.type != "finetune":
+                logger.warning(
```

Comment:
I could see a case where you have a fine-tuned model with an adapter that you load back and just use it for zero shot/few shot, so that's why I wanted to leave this as a warning. What do you think?

Reply:
Yup that's right, it would be fine-tuning a model through (trainer.type: finetune) and then using the same model path with a modified config (trainer.type: none).

I can raise a validation error for now!